contrib/win32/win32compat/console.c

```diff
@@ -1098,13 +1099,13 @@ ConMoveVisibleWindow(int offset)
 				error("GetConsoleScreenBufferInfo failed with %d", GetLastError());
 				return;
 			}
-			if (memcpy_s(&visibleWindowRect, sizeof(visibleWindowRect), &consoleInfo.srWindow, sizeof(visibleWindowRect))) {
-				error("memcpy_s failed: %d.", errno);
+			if ((r = memcpy_s(&visibleWindowRect, sizeof(visibleWindowRect), &consoleInfo.srWindow, sizeof(visibleWindowRect))) != 0) {
+				error("memcpy_s failed: %d.", r);
```

Comment:
1) error("memcpy_s failed, error: %d.", r);
2) This change has to be made at all the places?